test/emulation.spec.ts

```diff
@@ -408,4 +408,34 @@ describe('Emulation', () => {
       await page.emulateNetworkConditions(null);
     });
   });
+
+  describeFailsFirefox('Page.emulateCPUThrottling', function () {
+    it('should slow down execution', async () => {
+      const { page } = getTestState();
+
+      async function measure() {
+        const start = new Date();
+        await page.evaluate(() => {
+          let i = 0;
+          while (i < 1000) {
+            i++;
+          }
+        });
+
+        const end = new Date();
+        return +end - +start;
+      }
+
+      const baseline = await measure();
+      await page.emulateCPUThrottling(100);
```

Comment:
does it need to be reset in a finally block? if the test fails, I guess the throttling might apply to the next tests.

Reply:
Other tests for emulation features also don't reset afterwards. I think it's because the test runner will always create a new page for every test.